src/System.Private.Xml/src/project.json

```diff
@@ -27,7 +26,7 @@
         "System.IO.FileSystem.Primitives": "4.4.0-beta-24705-01",
         "System.Linq": "4.4.0-beta-24705-01",
         "System.Net.Http": "4.4.0-beta-24705-01",
-        "System.Net.Primitives": "4.4.0-beta-24705-01",
+        "System.Net.Requests": "4.4.0-beta-24705-01",
```

Comment:
What causes a dependency on Requests? We just did work recently to break this dependency and move it to just Primitives so I would like to avoid regressing that if possible.